### PR TITLE
Switch from `fiware/orion-ci:deb` to `telefonicaiot/fiware-orion:ci`

### PR DESCRIPTION
**Pull request by mapedraza**
This PR changes in the code base all references from `fiware/orion-ci:deb` to `telefonicaiot/fiware-orion:ci`

Additionaly, this PR removes `.github/workflows/ciimage.yml` since the CI image now is build by docker hub

### CODE REVIEW

File: ci/README.jp.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+このシステムは、新しい PR がマスターに到達するたびにマスターブランチから構築される `telefonicaiot/fiware-orion:ci` に基づいており、
 すべてのビルド依存関係がオンボードにあるクリーンな環境を提供します。この Docker のビルドに使用される Dockerfile は、
 `ci/deb` ディレクトリにあります。
 
-テスト対象の PR ブランチが変更されたため、`fiware/orion-ci:deb` は再構築されないことに注意してください。したがって、
+テスト対象の PR ブランチが変更されたため、`telefonicaiot/fiware-orion:ci` は再構築されないことに注意してください。したがって、
 新しいライブラリまたはベースシステムを必要とする機能を開発している場合は、そのようなライブラリまたはベースシステムを
 `ci/deb/build-dep.sh` および/または `Dockerfile` に追加する PR を*最初に*実行する必要があります。その PR がマスターに
-マージされ、`fiware/orion-ci:deb` が再構築されると (Docker Hub の https://hub.docker.com/r/fiware/orion-ci/builds
+マージされ、`telefonicaiot/fiware-orion:ci` が再構築されると (Docker Hub の https://hub.docker.com/r/fiware/orion-ci/builds
```
Comment (fgalan) [inline ci/README.jp.md:10]: Link should be updated

File: ci/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+The system is based on `telefonicaiot/fiware-orion:ci` which is built from master branch each time a new PR lands in master,
 providing a clean environment with all build dependencies onboard. The Dockerfile used to build this docker is available
 in the `ci/deb` directory.
 
-Note that `fiware/orion-ci:deb` is *not* rebuilt due to changes in the PR branch under test. Thus, if you are developing
+Note that `telefonicaiot/fiware-orion:ci` is *not* rebuilt due to changes in the PR branch under test. Thus, if you are developing
 a functionality that requires a new library or base system you need to do *first* a PR adding such library or base system
-to `ci/deb/build-dep.sh` and/or `Dockerfile`. Once that PR gets merged into master and `fiware/orion-ci:deb` gets rebuild
+to `ci/deb/build-dep.sh` and/or `Dockerfile`. Once that PR gets merged into master and `telefonicaiot/fiware-orion:ci` gets rebuild
 (checking progress in Dockerhub at: https://hub.docker.com/r/fiware/orion-ci/builds) your PR branch with the new
```
Comment (fgalan) [inline ci/README.md:10]: Link should be updated

Comment (mapedraza) [reply ci/README.md]: Done in https://github.com/telefonicaid/fiware-orion/pull/4391/commits/7b8dd0f82d9e26628b262298c07d718e9af3f5c9

File: ci/README.jp.md (hunk 1 above)
Comment (mapedraza) [reply ci/README.jp.md]: Done in https://github.com/telefonicaid/fiware-orion/pull/4391/commits/7b8dd0f82d9e26628b262298c07d718e9af3f5c9

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**